docs/user/openstack/kuryr.md

```diff
@@ -76,4 +81,4 @@ There are known limitations when using Kuryr SDN:
 * There is an amphora load balancer VM being deployed per OpenShift service with the default Octavia load balancer driver (amphora driver). If the environment is resource constrained it could be a problem to create a large amount of services as each one will create a new VM.
 * Depending on the Octavia version, UDP listeners are not supported. This means that OpenShift UDP services are not supported unless OpenStack version is Train or newer.
 * Depending on the Octavia version, there is a known limitation of Octavia not supporting listeners on different protocols (e.g., UDP and TCP) on the same port. Thus services exposing the same port for different protocols are not supported unless OpenStack version is Train or newer.
-* Due to the above UDP limitations of Octavia, Kuryr is forcing pods to use TCP for DNS resolution (`use-vc` option at `resolv.conf`) if the OpenStack version is not Train or newer. This may be a problem for pods running Go applications compiled with `CGO_DEBUG` flag disabled as that forces to use the `go` resolver that is only using UDP and is not considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a problem also for musl-based containers as it's resolver does not support `use-vc` option. This would include e.g., images build from `alpine`.
+* Due to the above UDP limitations of Octavia, Kuryr is forcing pods to use TCP for DNS resolution (`use-vc` option at `resolv.conf`) if the OpenStack version is not Train or newer. This may be a problem for pods running Go applications compiled with `CGO_ENABLED` flag disabled as that forces to use the `go` resolver that is only using UDP and is not considering the `use-vc` option added by Kuryr to the `resolv.conf`. This is a problem also for musl-based containers as it's resolver does not support `use-vc` option. This would include e.g., images build from `alpine`.
```

Comment:
`s/it's resolver/its resolver/`

Also, I'll bring up `CGO_ENABLED` again (sorry). Do you think we can be even more explicit, to avoid misunderstandings? Like:

```plaintext
This may be a problem for Go applications compiled without CGO support (i.e. `CGO_ENABLED=0`) as the native Go resolver is using UDP only and is not considering... 
```

Reply:
After some research, I've found something potentially interesting.

The `net` package [defaults](https://golang.org/pkg/net/#hdr-Name_Resolution) to using the native resolver. You might need `GODEBUG=netdns=cgo` to force it to use the libc resolver (provided that `CGO_ENABLED=0` has not been passed).

However, that should not matter (any more), because with Go 1.13, the native resolver should [use DNS over TCP when use-vc is set in resolv.conf](https://go-review.googlesource.com/c/go/+/156366/).

Unless your experience disproves this claim?

Reply:
thanks!